alacritty/src/event.rs

```diff
@@ -354,14 +352,6 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         let mut env_args = env::args();
         let alacritty = env_args.next().unwrap();
 
-        // Use working directory of controlling process, or fallback to initial shell, then add it
-        // as working-directory parameter.
-        #[cfg(unix)]
-        let mut args = foreground_process_path()
-            .map(|path| vec!["--working-directory".into(), path])
-            .unwrap_or_default();
-
-        #[cfg(not(unix))]
         let mut args: Vec<PathBuf> = Vec::new();
 
         let working_directory_set = !args.is_empty();
```

Comment:
You're breaking the behavior this boolean is used for by removing the `args` assignment on unix. See the following lines of code here.

Reply:
Thanks, I think it is fixed (simply skipping `--working-directory arg` now).